common/containers/Tabs/SendTransaction/components/ConfirmationModal.jsx

```diff
@@ -16,12 +18,14 @@ type Props = {
   rawTransaction: RawTransaction,
   wallet: BaseWallet,
   node: NodeConfig,
+  token: ?Token,
   onConfirm: RawTransaction => void,
   onCancel: () => void
 };
 
 type State = {
```

Comment:
Moved most of this out of state, and into a `_decodeTransaction` function:  a8c579b4b7de6136d9102aa3f4887182e6c3d6d8